erpnext/controllers/queries.py

```diff
@@ -550,50 +550,32 @@ def item_manufacturer_query(doctype, txt, searchfield, start, page_len, filters)
 @frappe.whitelist()
 def get_purchase_receipts(doctype, txt, searchfield, start, page_len, filters):
 	fields = get_fields("Purchase Receipt", ["name"])
+	item_filters = [
+		['Purchase Receipt', 'docstatus', '=', '1', 'and', 'name', 'like', frappe.db.escape('%{0}%'.format(txt))],	
+		['Purchase Receipt Item', 'item_code', '=', filters.get("item_code"), 'and', 'parent', '=', fields[0]]
+	]
```

Comment:
What is this query ?
```suggestion
	item_filters = [
		['Purchase Receipt', 'docstatus', '=', '1'],
		['Purchase Receipt', 'name', 'like', frappe.db.escape('%{0}%'.format(txt))],	
		['Purchase Receipt Item', 'item_code', '=', filters.get("item_code")],
		['Purchase Receipt Item', 'parent', '=', fields[0]]
	]
```